src/software/ai/evaluation/scoring/skills/feasibility_visitor.cpp

```diff
@@ -32,44 +32,30 @@ void FeasibilityVisitor::visit(const KeepAwaySkill& skill)
 
 void FeasibilityVisitor::visit(const PassSkill& skill)
 {
-    const std::vector<OffenseSupportType>& committed_strategy =
-        (*strategy_)->getCommittedOffenseSupport();
-
-    if (std::find(committed_strategy.begin(), committed_strategy.end(),
-                  OffenseSupportType::PASS_RECEIVER) == committed_strategy.end())
-    {
-        current_feasibility_ = 0;
-        return;
-    }
-
     PassWithRating best_pass = (*strategy_)->getBestUncommittedPass();
-
-    double min_score =
-        (*strategy_)->getAiConfig().shoot_or_pass_play_config().abs_min_pass_score();
-
-    if (best_pass.rating > min_score)
-    {
-        // TODO(arun): should we normalize this?
-        current_feasibility_ = best_pass.rating;
-        return;
-    }
-
-    current_feasibility_ = 0;
-    return;
+    current_feasibility_ = normalizeValueToRange(best_pass.rating, 0.0, 0.7, 0.0, 1.0);
```

Comment:
What is the purpose or why is this change needed? Just curious about how it works.

Reply:
Not sure 😄 I have PassSkill disabled locally so I forgot I changed this. It will be rewritten in the future